src/app/modules/react-migration/react-migration.controller.ts

```diff
@@ -0,0 +1,118 @@
+import path from 'path'
+
+import { FormAuthType } from '../../../../shared/types'
+import config from '../../config/config'
+import { ControllerHandler } from '../core/core.types'
+import * as FormService from '../form/form.service'
+import * as PublicFormController from '../form/public-form/public-form.controller'
+import { RedirectParams } from '../form/public-form/public-form.types'
+import * as HomeController from '../home/home.controller'
+
+const RESPONDENT_COOKIE_NAME = 'v2-respondent-ui'
+const ADMIN_COOKIE_NAME = 'v2-admin-ui'
+
+export type SetEnvironmentParams = {
+  ui: 'react' | 'angular'
+}
+
+export const RESPONDENT_COOKIE_OPTIONS = {
+  httpOnly: true,
+  sameSite: 'strict' as const,
+  secure: !config.isDev,
+}
+
+export const ADMIN_COOKIE_OPTIONS = {
+  httpOnly: true,
+  maxAge: 365 * 24 * 60 * 60, // 1 year
```

Comment:
Reduce to 2 months

Reply:
https://github.com/opengovsg/FormSG/pull/3750/commits/67e50620156ad0c043c62834ae9ac4fb18d5f415